### DISCUSSION TITLE
My LTBEEF doesnt work but i have it on for now

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#893] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So the reason why it doesnt work is because my chrome-extension://knneimmpeamikmjijmnhjmmobddleohj/_generated_background_page.html doesn't load does this mean I cant do it anymore?

**Comment by PenguinLord561** [comment]
@CGRmags hmmm... Does  chrome-extension://knneimmpeamikmjijmnhjmmobddleohj/manifest.json work?

**Reply by CGRmags** [reply]
@PenguinLord561 It just hangs like the other link

**Reply by CGRmags** [reply]
@PenguinLord561  @AshtonDavies Do you have explanation to this?

**Reply by AshtonDavies** [reply]
It sounds like your extension is hanged.

**Reply by CGRmags** [reply]
@AshtonDavies Is that an outcome of doing LTBEEF flood or did my school do smth?

**Reply by AshtonDavies** [reply]
> @AshtonDavies Is that an outcome of doing LTBEEF flood or did my school do smth?

It would be an outcome of doing/timing something wrong. Don't forget to flip the switch in extension settings.

**Reply by CGRmags** [reply]
like switch it back? will it take away the unblocker?

**Reply by AshtonDavies** [reply, marked as answer]
> like switch it back? will it take away the unblocker?

Switching it either way changes a permission, therefore making it refresh the extension to get it following that setting. Refreshing it is necessary for filtering extensions that want Chrome to wait for a response before loading pages. That's why just hanging it will result in your tabs not loading.

**Comment by CGRmags** [comment]
@AshtonDavies  like switch it back? will it take away the unblocker?
